portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,10 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  align-items: center;
```

Comment:
order the css rules in alphabetic order
align-items:
margin-left:
margin-right:
width: